pages/linux/e4defrag.md

```diff
@@ -0,0 +1,15 @@
+# e4defrag
+
+> Defragment an ext4 file system.
+
+- Defragment the file system:
+
+`e4defrag {{/dev/sda1}}`
```

Comment:
Based on the [man page of mkfs.ext4](https://linux.die.net/man/8/mkfs.ext4) and the [ArchWiki](https://wiki.archlinux.org/index.php/Ext4#Disabling_journaling) I recommend the use of:
```suggestion
`e4defrag {{/dev/sdXN}}`
```
to avoid run this command over a real partition by accident when the user copy and paste this example